### ISSUE TITLE
Support rich markup in version strings

### ISSUE THREAD

**Issue by glenfletcher** [BrianPugh/cyclopts#187]
While rich markup is supported by help messages, it is not used when printing the version instead, it's just a straight call to print https://github.com/BrianPugh/cyclopts/blob/2fcef8f9af7e49c4c33e62d60df26a64577f0725/cyclopts/core.py#L385-L387

I was able to work around this using the following based on `help_print` to override `version_print`:

```py
class App(cyclopts.App):
    def version_print(self) -> None:
        """Print the application version."""
        console = self._resolve_console(None, None)
        console.print(self.version() if callable(self.version) else self.version)
```

from `help_print`, I just used the defaults for parameters `tokens` and `console` from `help_print`'s singuature:
https://github.com/BrianPugh/cyclopts/blob/2fcef8f9af7e49c4c33e62d60df26a64577f0725/cyclopts/core.py#L849-L856

**Comment by BrianPugh** [comment]
thanks for the report! I'll open up a PR integrating these changes.

**Comment by BrianPugh** [comment]
resolved in #188
